packages/forma-36-react-components/src/components/Autocomplete/Autocomplete.css

```
@@ -0,0 +1,21 @@
+@import 'settings/colors';
+@import 'settings/typography';
+@import 'settings/transitions';
+
+.autocompleteInput {
+  display: flex;
+}
+
+.autocompleteInput input::-webkit-search-cancel-button {
+  -webkit-appearance: none;
+}
+
+.autocompleteInput input::-ms-clear {
+  display: none;
+}
+
+.inputIconButton {
+  position: absolute;
+  top: calc(50% - 9px);
```

I think this would work and means we're using our tokens for spacing

```suggestion
  top: calc(50% - var(--spacing-xs));
```